moped-editor/src/layouts/DashboardLayout/Footer.js

```diff
@@ -13,13 +13,32 @@ const useStyles = makeStyles(theme => ({
 
 const Footer = () => {
   const classes = useStyles();
+  const [mostRecentTag, setMostRecentTag] = useState(`v${pckg.version}`); // use default tag
+
+  useEffect(() => {
+    getMostRecentGithubTags();
+  });
+
+  /**
+   * Async function that updates the mostRecentTag state based on API callto Github "tag" endpoint
+   */
+  const getMostRecentGithubTags = async () => {
+    const githubTagsApiEndpoint =
+      "https://api.github.com/repos/cityofaustin/atd-moped/tags";
```

Comment:
I think the "cityofaustin/atd-moped" portion of this string is specific to our particular deployment of the app and could be incorporated into the [configuration file](https://github.com/cityofaustin/atd-moped/blob/main/moped-editor/.env-cmdrc). Not a big deal by any means though.

Reply:
Yeah it felt a little bit like hard-coding to me, but I had a hard time thinking of how this Footer would be reused outside of this app, or our app changing repository names anytime soon, if ever.